backend/typescript/services/implementations/serviceRequest.ts

```diff
@@ -11,7 +11,28 @@ class ServiceRequest implements IServiceRequest {
 
   async getServiceRequestByID(
     requestId: string,
-  ): Promise<Prisma.getServiceRequestByID> {}
+  ): Promise<Prisma.getServiceRequestByID> {
+    try {
+      const serviceRequest: ServiceRequest | null = await Prisma.serviceRequest.findUnique({
+        where: {
+          ID: requestId,
```

Comment:
```suggestion
          id: requestId,
```
in our prisma model, id is lowercase